updater/Gemfile.lock

```diff
@@ -174,7 +174,7 @@ GEM
       reline (>= 0.3.8)
     diff-lcs (1.5.1)
     docile (1.4.0)
-    docker_registry2 (1.18.0)
```

Comment:
Although this https://github.com/dependabot/dependabot-core/blob/f238da5b58dabcf59014f8f6ec2409f6ea3804b6/common/dependabot-common.gemspec#L33 definition covers this change, we should consider upgrading there instead

Reply:
ok, I have pushed commits with the modification in `dependabot-core/common/dependabot-common.gemspec`, and also manually in `Gemfile.lock` and `updater/Gemfile.lock`.
Unsure about all that, notably about `Gemfile.lock`.
I'm not a ruby developer, I'm not familiar with the ruby packaging, but `docker_registry2` 1.18.0..1.18.2 has no dependency change (except dev dependency), so it should be ok to patch the Gemfile.lock manually.